app/server/core/config.ts

```diff
@@ -6,12 +6,16 @@ const envSchema = type({
 	SERVER_IP: 'string = "localhost"',
 	SERVER_IDLE_TIMEOUT: 'string.integer.parse = "60"',
 	RESTIC_HOSTNAME: "string = 'zerobyte'",
+	PORT: 'string.integer.parse = "4096"',
+	"MIGRATIONS_PATH?": "string",
```

Comment:
For consistency with the rest

```suggestion
	"MIGRATIONS_PATH": "string?",
```